PyMPDATA/solver.py

```diff
@@ -107,8 +109,12 @@ def advector(self) -> VectorField:
 
     @property
     def g_factor(self) -> ScalarField:
-        """g_factor field (with halo), unmodified by advance(),
-        assumed to be constant-in-time"""
+        """G_factor field (with halo), unmodified by advance(), assumed to be constant-in-time.
+        Can be used i.e. to transform the geometry of the advector
```

Comment:
the keywords "Jacobian" and "coordinate transformation" are missing here - would be much more precise. Also, since we have examples depicting it, likely worth to refer to it?

Reply:
Added both keywords and the reference to PyMPDATA implementation of Williamson & Rasch 1989